src/Etg.Yams.Core/Deploy/RemoteApplicationDeploymentDirectory.cs

```diff
@@ -35,7 +35,7 @@ public RemoteApplicationDeploymentDirectory(IDeploymentRepository deploymentRepo
                     {
                         if (!await _deploymentRepository.HasApplicationBinaries(appIdentity))
                         {
-                            Trace.TraceError($"Could not binaries for application {appIdentity} in the yams repository");
+                            Trace.TraceError($"Could not find binaries for application {appIdentity} of {version} in the yams repository");
```

Comment:
AppIdentity already contains the version and ToString prints both the id and the version.